onnx2torch/node_converters/squeeze.py

```diff
@@ -9,44 +9,60 @@
 import torch
 import torch._C as torch_C
 from torch import nn
+from torch.onnx.symbolic_helper import _export_onnx_opset_version as opset_version
 
 from onnx2torch.node_converters.registry import add_converter
 from onnx2torch.onnx_graph import OnnxGraph
 from onnx2torch.onnx_node import OnnxNode
 from onnx2torch.utils.common import OnnxMapping
-from onnx2torch.utils.common import OnnxToTorchModule
 from onnx2torch.utils.common import OperationConverterResult
 from onnx2torch.utils.common import get_const_value
 from onnx2torch.utils.common import onnx_mapping_from_node
 from onnx2torch.utils.custom_export_to_onnx import CustomExportToOnnx
 from onnx2torch.utils.custom_export_to_onnx import OnnxToTorchModuleWithCustomExport
 
-
-class OnnxSqueezeStaticAxes(nn.Module, OnnxToTorchModule):
+class OnnxSqueezeStaticAxes(nn.Module, OnnxToTorchModuleWithCustomExport):
 
     def __init__(self, axes: Optional[List[int]] = None):
         super().__init__()
-        if axes is not None:
+        if not self.is_empty_axes(axes):
             axes = sorted(axes, reverse=True)
 
         self.axes = axes
 
-    def forward(self, input_tensor: torch.Tensor) -> torch.Tensor:
-        if not self.axes:
+    @staticmethod
+    def is_empty_axes(axes: List[int]) -> bool:
+        return axes is None
```

Comment:
in this case `is_empty_axes` function is not necessary

```python
if not self.axes:
    ...
```
will be enough

Reply:
Done